doc/metrics: improve wording for `project` parameter (#15914)
canonical/lxd: doc/metrics.md

  # to the `/var/snap/lxd/common/lxd/cluster.crt` file found on every member of
  # the LXD cluster.
  #
  # Note: When using a certificate restricted to multiple projects,
  #       use the `project` param to only scrape a specific project or projects.
  #       Otherwise, omit it to return the metrics for all the accessible
  #       projects in one scrape.
  #

>   # Note: each member of the cluster only provide metrics for instances it runs

Reviewer: ```suggestion
  # Note: Each member of the cluster only provides metrics for its local instances.
```

>   #       locally this is why the `lxd-hdc` cluster lists 3 targets

Reviewer: ```suggestion
  #       This is why the `lxd-hdc` cluster lists 3 targets.
```
